package.json

```diff
@@ -156,9 +171,11 @@
     "webpack-bundle-analyzer": "^4.5.0",
     "webpack-cli": "^4.9.2",
     "webpackbar": "^5.0.2",
-    "worker-loader": "^3.0.8"
+    "worker-loader": "^3.0.8",
+    "workspaces-run": "^1.0.1"
   },
   "scripts": {
+    "bundle": "rollup",
```

Comment:
Shouldn't this be `rollup -c` ?

Reply:
Haven‘t tested it again recently, but I thought Rollup automatically looks for the default config so there‘s no need to specify it here.

Reply:
Okay, looks like it doesn't work without it.
![image](https://user-images.githubusercontent.com/6297436/151694404-212870d3-d757-4ec7-af48-f08cc7cb08fb.png)

Reply:
Oh, looks like we need to use `rollup -c`. Then it will default to `rollup.config.js`. Unexpected.